src/adapter.js

```diff
@@ -47,6 +47,19 @@
             System.bundles = [];
         }
 
+        var systemInstantiate = System.instantiate;
```

Comment:
@maxwellpeterson-wf Not a problem, I'll add the comments, but I'll also leave you with a quick explanation for now. That code is converting the paths from a system ('file://') format into urls. Including `/base/` is just one aspect of that, and it should probably not be hard coded. This part is necessary because the Istanbul instrumentation happens on the system, but the execution happens on the browser. In other words, say file "x" is on your system. Before Karma launches the browsers, file "x" is [prepossessed and instrumented by Istanbul](https://github.com/Workiva/karma-jspm/pull/147/files#diff-e3f8f67a789d4aa4023fa8b987fc4cacR67), we then take that source and [store](https://github.com/Workiva/karma-jspm/pull/147/files#diff-e3f8f67a789d4aa4023fa8b987fc4cacR66) in an object for the browser to [access](https://github.com/Workiva/karma-jspm/pull/147/files#diff-e52a9a67956bd51d044af4058fb939ceR58). But when the  browser gets it, all the "urls" are system paths (i.e. file:///home/project/x.js). So we first need to [extract](https://github.com/Workiva/karma-jspm/pull/147/files#diff-e52a9a67956bd51d044af4058fb939ceR55) only the unique portion (i.e "project/x.js" - the `fileKey`) to retrieve the source of the file from the object and then [convert](https://github.com/Workiva/karma-jspm/pull/147/files#diff-e52a9a67956bd51d044af4058fb939ceR58) the paths in the source's content, whose dependencies still has it's "urls" in system file format, to actual urls (i.e. http://localhost:8080/base/project/x.js). I hope that clarifies a bit.